integration-tests/smoke/vrfv3_test.go

```diff
@@ -0,0 +1,336 @@
+package smoke
+
+import (
+	"fmt"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	eth "github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+	ocr2vrftypes "github.com/smartcontractkit/ocr2vrf/types"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func TestVRFv3Basic(t *testing.T) {
+	linkEthFeedResponse := big.NewInt(1e18)
+	t.Parallel()
+	testEnvironment, testNetwork := setupVRFv3Test(t)
+
+	chainClient, err := blockchain.NewEVMClient(testNetwork, testEnvironment)
+	require.NoError(t, err)
+	contractDeployer, err := contracts.NewContractDeployer(chainClient)
+	require.NoError(t, err)
+	chainlinkNodes, err := client.ConnectChainlinkNodes(testEnvironment)
+	require.NoError(t, err)
+	nodeAddresses, err := actions.ChainlinkNodeAddresses(chainlinkNodes)
+	require.NoError(t, err, "Retreiving on-chain wallet addresses for chainlink nodes shouldn't fail")
+	//t.Cleanup(func() {
+	//	err := actions.TeardownSuite(t, testEnvironment, ctfutils.ProjectRoot, chainlinkNodes, nil, chainClient)
+	//	require.NoError(t, err, "Error tearing down environment")
+	//})
+	chainClient.ParallelTransactions(true)
+
+	//1. DEPLOY LINK TOKEN
+	linkToken, err := contractDeployer.DeployLinkTokenContract()
+	require.NoError(t, err)
+
+	//2. DEPLOY ETHLINK FEED
+	mockETHLinkFeed, err := contractDeployer.DeployMockETHLINKFeed(linkEthFeedResponse)
+	require.NoError(t, err)
+
+	//ethLinkFeedAddress := "0xb4c4a493AB6356497713A78FFA6c60FB53517c63"
+
+	//3. Deploy DKG contract
+	dkg, err := contractDeployer.DeployDKG()
+	require.NoError(t, err)
+
+	err = chainClient.WaitForEvents()
+	require.NoError(t, err)
+
+	beaconPeriodBlocksCount := big.NewInt(3)
+	//4. Deploy VRFCoordinator(beaconPeriodBlocks, linkAddress, linkEthfeedAddress)
+	coordinator, err := contractDeployer.DeployVRFCoordinatorV3(beaconPeriodBlocksCount, linkToken.Address(), mockETHLinkFeed.Address())
+	require.NoError(t, err)
+	err = chainClient.WaitForEvents()
+	require.NoError(t, err)
+
+	//5. Deploy VRFBeacon
+	//keyId can be any random value
+	keyId := "aee00d81f822f882b6fe28489822f59ebb21ea95c0ae21d9f67c0239461148fc"
```

Comment:
should we move this and other constants (e.g. `beaconPeriodBlocksCount`) to the top of the file so that we can look at all configs at a glance?